static/images/diagrams/cloud_ssh_workflow.txt

```diff
@@ -0,0 +1,41 @@
+# Use http://sequencediagram.org/ to generate image
+
+title Cloud-based provisioning with Puppet over ssh
+
+participant User
+participant "Provisioned Instance" as Host
+participant "Foreman" as Foreman
+participant "Foreman Proxy" as Proxy
+participant "Infrastructure Cloud" as Cloud
+participant DNS
+participant "Puppet master" as Puppet
+
+linear
+autonumber
+
+note over Foreman:has an Image with credentials defined
```

Comment:
Hmmm how about `note over Foreman:has an Image with credentials defined in Compute Resource`